app/posts/apply-for-teacher-training/2022-11-07-changing-how-we-ask-about-disabilities-and-health-conditions.md

```diff
@@ -0,0 +1,70 @@
+date: 2022-11-09
+ogImage:
+  src: /apply-for-teacher-training/changing-how-we-ask-about-disabilities-and-health-conditions/disability-question-changes.png
+  alt: "Illustration showing that previously there was an ‘Are you disabled?’ question which led to a ’What disabilities do you have?’ question, and now there is a single ‘Do you have any of these disabilities or health conditions?’ question"
+
+---
+
+![Illustration showing that previously there was an ‘Are you disabled?’ question which led to a ’What disabilities do you have?’ question, and now there is a single ‘Do you have any of these disabilities or health conditions?’ question](disability-question-changes.png)
+
+We’ve changed the way we ask candidates about disabilities and health conditions in the equality and diversity monitoring questions shown just before submitting an application.
+
+These changes have resulted in many more people selecting a disability or health condition.
+
+## The issue
+
+Previously, we asked an initial question of ’Are you disabled?’ with the options ’Yes’, ‘No’ and ‘Prefer not to say’.
+
+![Screenshot showing page asking ’Are you disabled?’ with the options ’Yes’, ‘No’ and ‘Prefer not to say’](are-you-disabled.png "Previous design for filter question")
+
+If a candidate selected Yes, then we presented a list of checkboxes, with the question ’What disabilities do you have?’
+
+![Screenshot showing page with the question ’What disabilities do you have?’ and this list of options: Blind or a serious visual impairment not corrected by glasses, Dear or a serious hearing impairment, Learning difficulty for example dyslexia, Long-standing illness for example cancer, Mental health condition for example depression, Physical disability or mobility issue for example impaired use of arms or legs, Social or communication impairment for example Asperger’s and Other.](disabilities-before.png "Previous design for disabilities question")
+
+The main issue with presenting the questions in this way is that some candidates who might have selected one or more of the checkboxes may not have done so as they do not consider themselves disabled. We had some evidence of this from user research sessions, where one user who was dyslexic commented that not everyone considers this a disability.
+
+In addition, for the 2022 to 2023 recruitment cycle, the Higher Education Statistics Authority updated their recommended list of options to present.
+
+## What we changed
+
+![Screenshot showing page with the question ’Do you have any of these disabilities or health conditions?’](disabilities-and-health-conditions.png "Updated design for disabilities or health conditions question")
+
+We removed the first question, and instead present all users with a list of checkboxes.
+
+We changed the question to ‘Do you have any of these disabilities or health conditions?’ and added a ‘I do not have any of these disabilities or health conditions’ option.
```

Comment:
Could possibly mention that the negative options are mutually exclusive from the others.